src/config/assetsTransformer.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
```

Comment:
What is this file for?

Reply:
It used for parsing CSS and SCSS files. Please refer to line 71 of our package.json file where we used it.
And it helped sort out our parsing errors with SCSS